config.py

```diff
@@ -0,0 +1,5 @@
+def init():
```

Comment:
Would it not make sense to just throw this in the current `settings.py`? I don't necessarily like how it's just one file that is evaluated once on import but that is currently where the rest of the configs exist. I like having 1 place for all the config and this is something that we may eventually want to make a setting exposed via the QT app so keeping it with the rest makes sense.

Reply:
Nah it does make sense. I didn't do my due diligence and went back thru the src to see if i can place it somewhere existing already since I was trying to wrap up dinner lol. I'll shift it over to that. We may need to consider doing a global configuration settings tab for regenerate the cc key secret setting this option and other future configurations/toggles like the http randomizer.